parmed/rdkit/rdkit.py

```diff
@@ -32,10 +32,22 @@ def load(rmol):
         return PDBFile.parse(fh)
 
     @staticmethod
-    def from_smiles(smiles):
+    def from_smiles(smiles, coordinates=True):
         """
         Load smiles string to :class:`Structure`
+
+        Parameters
+        ----------
+        smiles : str, smiles
+        coordinates : bool, default True
+            if True, use `rdkit.Chem.AllChem.EmbedMultipleConfs to assign coordinates
+            if False, coordinates for all atoms are all 0.0
```

Comment:
Lack of coordinates is typically None rather than all zeroes. That's a much easier test for coordinates. Any reason not to do that here?